docs/helm.md

```diff
@@ -10,7 +10,7 @@ Monokle has built-in support for [Helm](https://helm.sh/) - a popular tool for m
 
 ## Helm Navigation
 
-When selecting a folder containing Helm charts (identified by Chart.yaml files), these will automatically be displayed in 
+When selecting a folder containing Helm charts (identified by values.yaml files), these will automatically be displayed in 
```

Comment:
this should be Chart.yaml file - perhaps "identified by the existence of a Chart.yaml files"

Reply:
I have replaced values.yaml files with Chart.yaml file.